src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/FlipViewPages/FlipView_TemplateSelectorPage.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<Page
+    x:Class="Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.FlipViewPages.FlipView_TemplateSelectorPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls.FlipViewPages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Page.Resources>
+		<ResourceDictionary>
+			<DataTemplate x:Key="ItemTemplatePair">
+				<Grid x:Name="GridPair"
+					  x:FieldModifier="public"
```

Comment:
no point in marking them public, right? not like we can access instances of that template as a "class"

Reply:
Thanks! Fixed. Removed the whole name as we are not gonna use it after all.